examples/Bremsstrahlung/pymodules/energyHistogram.py

```diff
@@ -0,0 +1,227 @@
+import sys
+import os
+import inspect
+
+import numpy as np
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+import h5py
+
+# prepare imports
+cmd_subfolder = os.path.realpath(
+    os.path.abspath(
+        os.path.join(
+            os.path.split(
+                inspect.getfile(
+                    inspect.currentframe()
+                )
+            )[0], ".."
+        )
+    )
+)
+if cmd_subfolder not in sys.path:
+    sys.path.insert(0, cmd_subfolder)
+
+
+# custom matplotlib settings
+params = {
+    'font.size': 20,
+    'xtick.labelsize': 20,
+    'ytick.labelsize': 20,
+    'figure.figsize': [12, 8],
+    'legend.fontsize': 20,
+    'legend.frameon': "False",
+    'legend.markerscale': 2,
+    'legend.numpoints': 1,
+    'lines.linestyle': '',
+    'lines.marker': "+",
+    'lines.markeredgewidth': 2,
+    'lines.markersize': 6
+}
+
+# overwrite matplotlib defaults
+mpl.rcParams.update(params)
+
+
+class energyHistogram:
+    def __init__(
+            self,
+            species_name="ph",
+            sim="./",
+            period=1000,
+            bin_count=1024,
+            min_energy=10,
+            max_energy=10000,
+            distance_to_detector=0,
+            slit_detector_x=None,
+            slit_detector_z=None):
+        """
+        species_name
```

Comment:
I am ok with both classical PEP 257 and PEP 287 for rst.

But you are right, this one ain’t either one nor the other :D